### DISCUSSION TITLE
Move Search Input of Table Builder

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#9550] [Help]
Is it possible to move the "search" entry from the table? I would like to know if it is possible to move the search field that is integrated into the Table Builder, I would like to align it to the left and not to the right as it is now. Thanks for the support.

![imagen](https://github.com/filamentphp/filament/assets/13526704/f62ffdd1-9a01-454b-8af6-ea85ee4d7c40)

**Comment by tudorcrisan1231** [comment]
hello, I dont think that they provide this feature by default, but I found a way to do it easily
1. you need to have a custom theme installed in the panel
2. add some css to search container

```
.fi-ta-header-toolbar{
    @apply flex-row-reverse;
}

.fi-ta-header-toolbar > *:nth-child(2){
    width: 100%;
    justify-content: space-between;
}
```

and that's it
![Screenshot 2024-11-08 at 13 04 47](https://github.com/user-attachments/assets/ccbac846-50c7-4839-ab11-96d187295e76)

**Comment by mstaack** [comment]
is this still working? cant get it to move to the left. added custom theme and this snippet. any hints?
